From 31f77e5e1aaf56b6a5bb04d4c9b9efc6cd578fab Mon Sep 17 00:00:00 2001
From: Erik Schumacher <erik.schumacher@iris-sensing.com>
Date: Tue, 20 Feb 2024 11:56:08 +0100
Subject: [PATCH 11/15] swupdate-lualoader: Set SECTION to devel/lua

Having an empty SECTION = "" causes packaging failures with rpm:
| DEBUG: Executing shell function BUILDSPEC
| error: line 6: Empty tag: Group:

Fix packaging by setting a SECTION

Signed-off-by: Erik Schumacher <erik.schumacher@iris-sensing.com>

Upstream-Status: Backported from Scarthgap to Mickledore
---
 recipes-lua/swupdate-lualoader/swupdate-lualoader_1.0.bb | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/recipes-lua/swupdate-lualoader/swupdate-lualoader_1.0.bb b/recipes-lua/swupdate-lualoader/swupdate-lualoader_1.0.bb
index d6886c7..044cad2 100644
--- a/recipes-lua/swupdate-lualoader/swupdate-lualoader_1.0.bb
+++ b/recipes-lua/swupdate-lualoader/swupdate-lualoader_1.0.bb
@@ -1,6 +1,6 @@
 SUMMARY = "Loader for swupdate"
 LICENSE = "MIT"
-SECTION = ""
+SECTION = "devel/lua"
 DEPENDS = "lua"
 LIC_FILES_CHKSUM = "file://${WORKDIR}/swupdate_handlers.lua;md5=354cf4af377edd962d2e8d78085d3ed7;beginline=1;endline=19"
 
-- 
2.45.1

